docs/05-technical-reference/00-architecture/svls-01-architecture.md

```diff
@@ -12,7 +12,7 @@ Serverless relies heavily on Kubernetes resources. It uses [Deployments](https:/
 
 1. Create a Function either through the UI or by applying a Function custom resource (CR). This CR contains the Function definition (business logic that you want to execute) and information on the environment on which it should run.
 
-2. Before the Function can be saved or modified, it is first updated and then verified by the [defaulting and validation webhooks](../svls-07-supported-webhooks.md) respectively.
+2. Before the Function can be saved or modified, it is first updated and then verified by the defaulting and validation webhooks respectively.
```

Comment:
Since the supported webhooks doc has been removed, do we still need this sentence?
Maybe we should keep the supported webhooks doc?

Reply:
So maybe instead of preparing a doc we can link to the [Kubernetes doc on webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)?